client/app/bundles/course/group/pages/GroupShow/GroupManager/GroupUserManagerList.jsx

```diff
@@ -107,49 +163,41 @@ const GroupUserManagerListItem = ({
 
       <ListItemText primaryTypographyProps={{ style: styles.listItemTextSize }}>
         {user.name}
+        {user.isPhantom ? <GhostIcon /> : ''}
+        {otherGroups ? ` (also a member of${otherGroups})` : ''}
```

Comment:
extract `also a member of` as intl

Reply:
this one, I have tried but somehow it produces weird error (it detects the id of the translation instead of its defaultMessages). I will look into it later

Reply:
done